panic: Unknown Resource Health

Observed when starting the `nnf-node-manager`.  This appears to have been introduced by #106.  I went ahead and rolled back to using the previous commit, dc59df3b26e18781d732351ac2e9d2d54c50273a, for now to avoid this.

```
time="2022-12-15T16:22:02-08:00" level=info msg="Initialize switch 0"
time="2022-12-15T16:22:02-08:00" level=info msg="identify switch 0"
time="2022-12-15T16:22:02-08:00" level=info msg="Identify Switch 0: Device ID: 1"
time="2022-12-15T16:22:02-08:00" level=info msg="Identify Switch 0: Device ID: 0"
time="2022-12-15T16:22:02-08:00" level=info msg="Identify Switch 0: Loading Mfg Info"
2022-12-15T16:22:02.229-0800    INFO    Starting workers    {"controller": "nnfnodeecdata", "controllerGroup": "nnf.cray.hpe.com", "controllerKind": "NnfNodeECData", "worker count": 1}
2022-12-15T16:22:02.229-0800    INFO    Starting workers    {"controller": "clientmount", "controllerGroup": "dws.cray.hpe.com", "controllerKind": "ClientMount", "worker count": 128}
2022-12-15T16:22:02.229-0800    INFO    Starting workers    {"controller": "nnfnodestorage", "controllerGroup": "nnf.cray.hpe.com", "controllerKind": "NnfNodeStorage", "worker count": 1}
2022-12-15T16:22:02.229-0800    INFO    Starting workers    {"controller": "nnfnode", "controllerGroup": "nnf.cray.hpe.com", "controllerKind": "NnfNode", "worker count": 1}
time="2022-12-15T16:22:02-08:00" level=info msg="Switch 0 identified: PAX 0"
time="2022-12-15T16:22:02-08:00" level=info msg="Initialize switch 1"
time="2022-12-15T16:22:02-08:00" level=info msg="identify switch 1"
time="2022-12-15T16:22:02-08:00" level=info msg="Identify Switch 1: Device ID: 1"
time="2022-12-15T16:22:02-08:00" level=info msg="Identify Switch 1: Loading Mfg Info"
2022-12-15T16:22:02.233-0800    INFO    Observed a panic in reconciler: Unknown Resource Health     {"controller": "nnfnode", "controllerGroup": "nnf.cray.hpe.com", "controllerKind": "NnfNode", "NnfNode": {"name":"nnf-nlc","namespace":"hetchy44"}, "namespace": "hetchy44", "name": "nnf-nlc", "reconcileID": "16e06921-e2cf-4470-a71b-163958611f84"}
panic: Unknown Resource Health  [recovered]
    2022-12-15T16:22:02.236508119-08:00 panic: Unknown Resource Health

goroutine 738 [running]:
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Reconcile.func12022-12-15T16:22:02.236517036-08:00 ()
    /workspace/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:118 +0x1f4
panic(2022-12-15T16:22:02.236524932-08:00 {0x16f7560, 0xc0002c7c50})
    /usr/local/go/src/runtime/panic.go:884 +0x2122022-12-15T16:22:02.236532576-08:00
github.com/NearNodeFlash/nnf-sos/api/v1alpha1.ResourceHealth(...)
    /workspace/api/v1alpha1/nnf_resource_health_type.go:512022-12-15T16:22:02.236544089-08:00
github.com/NearNodeFlash/nnf-sos/controllers.(*NnfNodeReconciler).Reconcile(0xc000bb45a0, {0x1c4b6a8, 0xc000942210}, {{{0xc000048026, 2022-12-15T16:22:02.236551703-08:00 0x8}, {0x19a4b5f, 0x7}}})
    2022-12-15T16:22:02.236559108-08:00 /workspace/controllers/nnf_node_controller.go:226 +0xe85
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Reconcile(0x1c4b600?, 2022-12-15T16:22:02.236566472-08:00 {0x1c4b6a8?, 0xc000942210?}, {{{2022-12-15T16:22:02.236573656-08:00 0xc000048026?, 0x18c4d80?}, {0x19a4b5f?, 2022-12-15T16:22:02.236580749-08:00 0x4045d4?}}})
    /workspace/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:121 +2022-12-15T16:22:02.236588033-08:00 0xc8
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler(0xc0002a8dc0, {0x1c4b600, 0xc0009038802022-12-15T16:22:02.236595277-08:00 }, {0x17e3aa0?, 0xc000bc2040?})
    2022-12-15T16:22:02.236602862-08:00 /workspace/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:320 +0x33c
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem(0xc0002a8dc0, {2022-12-15T16:22:02.236610286-08:00 0x1c4b600, 0xc000903880})
    /workspace/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:273 +0x1d9
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func2.2()
    /workspace/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:234 +0x85
created by sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func2
    /workspace/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:230 +0x333
Stream closed EOF for nnf-system/nnf-node-manager-rjssf (manager)

--- a/go.mod
+++ b/go.mod
@@ -5,7 +5,7 @@ go 1.19
 require (
 	github.com/HewlettPackard/dws v0.0.0-20221110142740-abf44bba3b47
 	github.com/NearNodeFlash/lustre-fs-operator v0.0.0-20220929204230-5dcfe552c9e0
-	github.com/NearNodeFlash/nnf-ec v0.0.0-20221215195930-dde28910f6c3
+	github.com/NearNodeFlash/nnf-ec v0.0.0-20221216152313-0643bca404f9
 	github.com/ghodss/yaml v1.0.0
 	github.com/go-logr/logr v1.2.3
 	github.com/google/uuid v1.3.0

